### ISSUE TITLE
[Question]: Shouldn't the tfrs.examples.movielens.sample_listwise function sort according to rating?

### ISSUE THREAD

**Issue by fuchami** [tensorflow/recommenders#481]
Thanks for providing a great Listwise ranking tutorial.
https://www.tensorflow.org/recommenders/examples/listwise_ranking

After reading this tutorial and code here, I have a question.
In the `tfrs.examples.movielens.sample_listwise` function, shouldn't it sort according to the rating value? Is there any reason not to do this?

Sorry if my understanding of LTR is lacking.

**Comment by patrickorlando** [comment]
Hey @fuchami,

The `ListMLELoss()` function takes logits and labels representing the relevance of each item in the list. It doesn't depend on the order of the items in the list and so there is no need to sort the items by the user rating.
You could do it and the only reason not to do it is simply because it has no effect.

**Comment by fuchami** [comment]
Thanks @patrickorlando for your answer.
I understand now that reordering doesn't make sense.
